packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -590,11 +594,14 @@ class FlameChartUtils {
     // Node right with zoom and insets taken into consideration.
     final nodeRight =
         (node.rect.right - chartStartInset) * nodeZoom + chartStartInset;
-    return nextNode == null
+    final padding = nextNode == null
         ? chartWidth - nodeRight
         : ((nextNode.rect.left - chartStartInset) * nextNodeZoom +
                 chartStartInset) -
             nodeRight;
+    // Prevent negative padding from being returned. -0.0 can be returned from
+    // the above calculation due to double precision when nodes are very small.
```

Comment:
nit: use the same comment for this and the previous case.
bike shedding the comment, maybe write it as?
`// Floating point rounding error can result in slightly negative padding.`

Reply:
done